tilings/algorithms/subclass_verification.py

```diff
@@ -58,15 +59,24 @@ def quick_pare(self) -> Set[Perm]:
 
         return self.perms_to_check
 
-    def verified(self) -> bool:
+    @property
+    def subclasses(self) -> Tuple[Perm, ...]:
+        if self._subclasses is None:
+            self.compute_subclasses()
+        return cast(Tuple[Perm, ...], self._subclasses)
+
+    def compute_subclasses(self) -> None:
+
+        self._subclasses = tuple()
+
         # It is a waste of time to check a factorable tiling, since we will check its
         # children eventually.
-        if len(self.tiling.point_cells) > 0 or Factor(self.tiling).factorable():
-            return False
+        if Factor(self.tiling).factorable():
+            return
```

Comment:
Should this check happen on the strategy. We might want to add a `Tiling.subclass_verified` for example.